example/models/food/v1/food.pb.pgdb.go

```diff
@@ -1953,6 +2013,32 @@ func (x *PastaIngredientPBDataQueryType) Identifier() exp.IdentifierExpression {
 	return exp.NewIdentifierExpression("", x.tableName, x.column)
 }
 
+type PastaIngredientmodel_embeddings_MODEL_GPT_3_5_XXXQueryType struct {
+	column    string
+	tableName string
+}
+
+func (x *PastaIngredientDBQueryUnsafe) model_embeddings_MODEL_GPT_3_5_XXX() *PastaIngredientmodel_embeddings_MODEL_GPT_3_5_XXXQueryType {
+	return &PastaIngredientmodel_embeddings_MODEL_GPT_3_5_XXXQueryType{tableName: x.tableName, column: "pb$" + "model_embeddings_1"}
+}
+
+func (x *PastaIngredientmodel_embeddings_MODEL_GPT_3_5_XXXQueryType) Identifier() exp.IdentifierExpression {
+	return exp.NewIdentifierExpression("", x.tableName, x.column)
+}
+
+type PastaIngredientmodel_embeddings_MODEL_GPT_4_0_XXXQueryType struct {
+	column    string
+	tableName string
+}
+
+func (x *PastaIngredientDBQueryUnsafe) model_embeddings_MODEL_GPT_4_0_XXX() *PastaIngredientmodel_embeddings_MODEL_GPT_4_0_XXXQueryType {
```

Comment:
I think the name for this prob needs to be ctx.Name()?  becaue this function isn't exported right now?

Reply:
Also we should strings.TrimPrefix("MODEL_"* from the enum name i think?

Reply:
added this in https://github.com/ductone/protoc-gen-pgdb/pull/66/commits/fb6ee23cf16d8fef81bb4acfbe1aa59e70da9030